go/vt/vtctl/workflow/traffic_switcher.go

```diff
@@ -487,23 +487,32 @@ func (ts *trafficSwitcher) dropParticipatingTablesFromKeyspace(ctx context.Conte
 func (ts *trafficSwitcher) removeSourceTables(ctx context.Context, removalType TableRemovalType) error {
 	err := ts.ForAllSources(func(source *MigrationSource) error {
 		for _, tableName := range ts.Tables() {
-			query := fmt.Sprintf("drop table %s.%s",
-				sqlescape.EscapeID(sqlescape.UnescapeID(source.GetPrimary().DbName())),
-				sqlescape.EscapeID(sqlescape.UnescapeID(tableName)))
+			sanitizedPrimaryDbName, err := sqlescape.UnescapeID(source.GetPrimary().DbName())
+			if err != nil {
+				return err
+			}
+			primaryDbName := sqlescape.EscapeID(sanitizedPrimaryDbName)
+			sanitizedTableName, err := sqlescape.UnescapeID(tableName)
+			if err != nil {
+				return err
+			}
+			tableName := sqlescape.EscapeID(sanitizedTableName)
```

Comment:
Feels like a pattern here. You `UnescapeID` then `EscapeID`. Perhaps this should be a single function called something like `EnsureEscaped() (string, error)` which does both actions in a single call?

Then you'd do something like:
```go
primaryDbName, err := sqlescape.EnsureEscaped(source.GetPrimary().DbName())
...
tableName, err := sqlescape.EnsureEscaped(tableName)
```

Reply:
made the suggested changes